Source/Core/Core/HW/EXI/BBA/TAP_Win32.cpp

```diff
@@ -283,7 +284,7 @@ void CEXIETHERNET::TAPNetworkInterface::ReadThreadHandler(TAPNetworkInterface* s
       // IO should be pending.
       if (GetLastError() != ERROR_IO_PENDING)
       {
-        ERROR_LOG(SP1, "ReadFile failed (err=0x%X)", GetLastError());
+        ERROR_LOG_FMT(SP1, "ReadFile failed (err=0x{:X})", GetLastError());
```

Comment:
```suggestion
        ERROR_LOG_FMT(SP1, "ReadFile failed (err={:#X})", GetLastError());
```

Reply:
Amended. Changed it from `X` to `x` though, since `#X` would result in `0X...` as opposed to the more expected `0x...`

Reply:
Ah, gotcha, I _kinda_ thought that was the reason behind it - but then again, I'd opt for the consistency of using `#` in the specifier instead of `0x` outside every time. 👍